acceptance/tests/pxp-module-puppet/run_puppet.rb

```diff
@@ -0,0 +1,49 @@
+require 'pxp-agent/test_helper.rb'
+
+test_name 'C95972 - pxp-module-puppet run' do
+
+  step 'Ensure each agent host has pxp-agent running and associated' do
+    agents.each_with_index do |agent, i|
+      on agent, puppet('resource service pxp-agent ensure=stopped')
+      cert_dir = configure_std_certs_on_host(agent)
+      create_remote_file(agent, pxp_agent_config_file(agent), pxp_config_json_using_test_certs(master, agent, i + 1, cert_dir).to_s)
+      on agent, puppet('resource service pxp-agent ensure=running')
+      expect_file_on_host_to_contain(agent, logfile(agent), PXP_AGENT_LOG_ENTRY_ASSOCIATION_SUCCESS, seconds_to_wait = 60)
```

Comment:
@er0ck  - I named the parameter as it's passed. Declaring a variable at the top of the test and referring to it here seemed excessive.